packages/string/native-string-types.d.ts

```
@@ -0,0 +1,148 @@
+         * Returns a string with all whitespace removed from the end of a string.
+         */
+        trimEnd<T extends string = string>(this: T): TrimEnd<T>;
+
+        /**
+         * Returns a string with all whitespace removed from the start of a string.
+         */
+        trimStart<T extends string = string>(this: T): TrimStart<T>;
+    }
+}
+
+export {};
```

Biome (1.9.4)

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)